app/src/main/java/com/xinto/opencord/ui/widget/Message.kt

```diff
@@ -224,6 +225,19 @@ fun WidgetMessageAuthor(
                     }
             )
         }
+        if (isBot) {
+            ProvideTextStyle(MaterialTheme.typography.labelSmall) {
+                Surface(
+                    color = MaterialTheme.colorScheme.secondary,
+                    shape = MaterialTheme.shapes.extraSmall,
+                ) {
+                    Text(
+                        "BOT",
```

Comment:
This should be an enum for other tags. Also use parameter name
```kt
Text(
  modifier = Modifier.padding(horizontal = 4.dp),
  text = "BOT"
)
```

Reply:
enum for what
bot=bot property, system = author flag, webhook no clue

Reply:
k

Reply:
webhook you get from message.webhookId